lab-davide/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-davide",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Add a description, doesn't need to be long, I usually use the same one I put in the description field of the github repo.